sw/otbn/crypto/p384_isoncurve.s

```diff
@@ -315,8 +310,63 @@ p384_check_public_key:
   unimp
 
   _y_valid:
-  jal       x1, p384_isoncurve_check
+  /* Fill gpp registers with pointers to variables */
```

Comment:
Wasn't this P384 commit merged as part of https://github.com/zerorisc/expo/pull/109? It can be removed here, I think (and imo it's good practice anyway to separate the review of a few dozen lines of sensitive cryptographic assembly from the review of a 2.5K-line infrastructure PR)